cloud-img/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM gcr.io/cloudshell-images/cloudshell:latest
```

Comment:
please change the directory where Dockerfile file is stored from cloud-img to cloud-shell

Reply:
Okay, will do!